### PR TITLE
Adding new language for organisation plans

### LINKED ISSUE #1499: add different language for organization plans
signing up for an organisation plan needs slightly different language than that for an individual

### PR DESCRIPTION
**Pull request by tansaku**
fixes #1499

### CODE REVIEW

**Comment by tansaku** [comment]
initial round trip - we also need to look at:

* [ ] rest of language in template
* [ ] language in email
* [ ] think about level of logic in view --> move to presenter?  move strings to yml file?

File: db/schema.rb [hunk 1]
```diff
@@ -248,44 +249,44 @@
...
     t.datetime "updated_at"
-    t.string   "first_name",             limit: 255
-    t.string   "last_name",              limit: 255
+    t.string   "first_name"
+    t.string   "last_name"
     t.boolean  "display_email"
-    t.string   "slug",                   limit: 255
-    t.string   "youtube_id",             limit: 255
-    t.boolean  "display_profile",                    default: true
+    t.string   "youtube_id"
```
Comment (mattlindsey) [inline db/schema.rb]: Hi @marianmosley.  Did you mean to reverse 'youtube_id' and 'slug'?

File: db/migrate/20161221182758_remove_stripe_customer_from_user.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class RemoveStripeCustomerFromUser < ActiveRecord::Migration
   def change
-    remove_column :users, :stripe_customer
+    remove_column :users, :stripe_customer, :string
```
Comment (mattlindsey) [inline db/migrate/20161221182758_remove_stripe_customer_from_user.rb]: Is the last parameter a mistake (":string")?

Comment (tansaku) [inline db/migrate/20161221182758_remove_stripe_customer_from_user.rb]: no - it's needed to allow the migration to be reversible

File: db/schema.rb (hunk 1 above)
Comment (tansaku) [inline db/schema.rb]: why do you think it's reversed @mattlindsey ?

**Comment by tansaku** [comment]
before we merge this we really should:

* [ ] review all the rest of the text in the interface
* [ ] add tests to check the emails contain the right text

**Pull request merged**